### DISCUSSION TITLE
[Feature] lightweight alternative to react-email

### DISCUSSION THREAD

**Discussion by dotlambda** [immich-app/immich#14002] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

React-email currently contributes more than 500 MiB to the Immich server's `node_modules` through the `@next/...`, `react-email`, and `next` subdirectories. That seems unnecessarily large just to format some emails. Note that the entire `node_modules` is less than 900 MiB.
Should we switch to a lightweight alternative?
Does someone have experience with e.g. https://jsx.email/?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by martabal** [comment]
Yeah react-email is horribly huge.

Here's the list of largest server dependencies using `make prod` with commit 91065db3ffe70e276e37ac97a83a3ae52fbe5326:
``` bash
du -shc -- * | sort -rh
681M    total
149M    react-email
136M    next
69M     geo-tz
42M     @opentelemetry
33M     typeorm
20M     exiftool-vendored.pl
13M     fluent-ffmpeg
12M     rxjs
11M     swagger-ui-dist
11M     libphonenumber-js
11M     @react-email
11M     @babel
9.7M    esbuild
7.7M    prettier
7.6M    class-validator
6.7M    tailwindcss
6.7M    @nestjs
```
